desktop/cmp/viewmanager/dialog/SaveAsDialog.ts

```diff
@@ -79,15 +86,21 @@ const formPanel = hoistCmp.factory<SaveAsDialogModel>({
                                 height: 70
                             })
                         }),
-                        hbox(
-                            formField({
-                                field: 'isShared',
-                                label: 'Share?',
-                                labelTextAlign: 'left',
-                                omit: !model.parent.enableSharing,
-                                item: switchInput()
-                            })
-                        )
+                        formField({
+                            field: 'visibility',
+                            label: 'Visibility',
+                            labelTextAlign: 'left',
+                            omit: visOptions.length == 1,
+                            item: select({options: visOptions}),
+                            info: visInfo
+                        }),
+                        formField({
+                            field: 'isPinned',
+                            label: 'Pinned?',
+                            labelTextAlign: 'left',
+                            info: 'Show in your default menu?',
```

Comment:
Thinking about the use of the word "default" here-- maybe just "Show in your menu?"?